src/analog/utils/CommandBinaryBuffer.ts

```diff
@@ -209,11 +209,26 @@ interface IParameterEmpty {}
  * deviceParameters.REPORTING_DATA_INTERVAL = `1`.
  */
 interface IParameterReportingDataInterval {
+    /**
+     * Minimal interval for data sending from device (in seconds) for a special schedule.
+     * Real value = value + pseudo-random value which is not more than `255` * `4`.
+     */
+    specialSchedulePeriod: number,
```

Comment:
```suggestion
    specialSchedulePeriod: types.TUint8,
```
?

and other places

Reply:
fixed